migration/v0.30/apply/90-psp-violations.sh

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env bash
+
+ROOT="$(readlink -f "$(dirname "${0}")/../../../")"
+
+# shellcheck source=scripts/migration/lib.sh
+source "${ROOT}/scripts/migration/lib.sh"
+
+results_mult_uniq=""
+
+function set_violating_resources() {
+  results_mult_uniq=""
+  results=()
+  cluster="$1"
+
+  # Get violations for PSPs
+  violations=$(kubectl_do "$cluster" get constraints -o yaml | yq4 '[.items[] | select(.kind == "K8sPSP*") | .status.violations[]]')
```

Comment:
This is based on the gatekeeper audit right?
The audit by default runs every [10 minutes](https://github.com/elastisys/compliantkubernetes-apps/blob/main/config/config/common-config.yaml#L595). Is there any risk that it will not have run with the new config before we get to this step in the migration?

Reply:
I tried removing `component=gatekeeper-psp` which removes the psp constraints from the cluster and then re-installing them again. The violations seems to be populated immediately ~10-20 seconds.

Reply:
Ok, sounds good.